### PR TITLE
Add german/english language options

### PR DESCRIPTION
**Pull request by alexanderbarnhill**

### CODE REVIEW

File: frontend/src/assets/i18n/de.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
+{
+  "HOME": {
+    "TITLE": "Home",
```
Comment (markuswich) [inline frontend/src/assets/i18n/de.json:3]: Startseite

**Comment by markuswich** [comment]
Translations for keyword:
https://www.linguee.com/english-german/translation/keyword.html

The project description on studon uses Schlagwort, but we can also keep it in english.

**Comment by alexanderbarnhill** [comment]
> Translations for keyword:
> https://www.linguee.com/english-german/translation/keyword.html
> 
> The project description on studon uses Schlagwort, but we can also keep it in english.

Yeah I wasn't sure about that. Keyword isn't completely correct either. I figured I'd just leave it

File: frontend/src/assets/i18n/de.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  "DOCUMENTS": {
+    "AVAILABLE_DOCUMENTS": "Verfügbare Dokumente",
+    "SEARCH_FOR_DOCUMENTS": "Nach Dokumenten Suchen",
+    "FILENAME": "Dateiname",
+    "MY_KEYWORDS": "Meine Keywords",
+    "APPLY_KEYWORDS": "Keywords anwenden",
+    "FILE_TYPE": "Dateityp",
+    "CREATEDAT": "Erestellt am",
+    "EDITING_IS": "Bearbeiten ist",
+    "DRAG_DROP_DIALOG": "Packen und ziehen Sie Ihre Dateien hier hin um sie hochzuladen",
```
Comment (markuswich) [inline frontend/src/assets/i18n/de.json:21]: "Ziehen Sie ihre Dateien in dieses Feld um sie hochzuladen"

File: frontend/src/assets/i18n/de.json [hunk 3]
```diff
@@ -0,0 +1,55 @@
+{
+  "HOME": {
+    "TITLE": "Startseite",
+    "ADD_DOCUMENTS": "Dokumente hinzufügen",
+    "KEYWORDS": "Keywords",
+    "DOCUMENTATION": "Dokumentation",
+    "SERVER_STATUS": "Server Status",
+    "UP": "Aktiv",
+    "LANGUAGE": "Sprache",
+    "MOTTO": "Wilkommen bei InfiniTag - Der automatischen Tagging Software, betrrieben durch maschinelles lernen."
```
Comment (Dimfred) [inline frontend/src/assets/i18n/de.json:10]: betrieben typo

File: frontend/src/assets/i18n/de.json [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+  },
+  "DOCUMENTS": {
+    "AVAILABLE_DOCUMENTS": "Verfügbare Dokumente",
+    "SEARCH_FOR_DOCUMENTS": "Nach Dokumenten Suchen",
+    "FILENAME": "Dateiname",
+    "MY_KEYWORDS": "Meine Keywords",
+    "APPLY_KEYWORDS": "Keywords anwenden",
+    "SEARCH_KEYWORDS": "Nach Keywords suchen",
+    "FILE_TYPE": "Dateityp",
+    "CREATEDAT": "Erestellt am",
```
Comment (Dimfred) [inline frontend/src/assets/i18n/de.json:20]: Erstellt am typo

File: frontend/src/assets/i18n/de.json [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+    "DRAG_DROP_DIALOG": "Ziehen Sie ihre Dateien in dieses Feld um sie hochzuladen",
+    "BROWSE_FOR_FILES": "Dokumente durchsuchen"
+  },
+  "KEYWORDS": {
+    "KEYWORD_MODEL_SETTINGS": "Modell Einstellungen",
+    "ADD_DIMENSION": "Dimension hinzufügen",
+    "ADD_KEYWORD": "Keyword hinzufügen",
+    "UNCATEGORIZED_DIMENSIONS": "Dimensionen ohne Kategorie",
+    "UNCATEGORIZED_KEYWORDS": "Keywords ohne Kategorie",
+    "KEYWORD_MODELS": "Keyword Modelle",
```
Comment (Dimfred) [inline frontend/src/assets/i18n/de.json:31]: Schlüsselwort Modelle :DD

Comment (alexanderbarnhill) [reply frontend/src/assets/i18n/de.json:31]: No

**Pull request merged**